integration-tests/reorg/reorg_test.go

```diff
@@ -0,0 +1,218 @@
+package reorg
+
+//revive:disable:dot-imports
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+	uuid "github.com/satori/go.uuid"
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/cdk8s/blockscout"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver"
+	mockservercfg "github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver-cfg"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/reorg"
+	"github.com/smartcontractkit/chainlink-testing-framework/actions"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/client"
+	"github.com/smartcontractkit/chainlink-testing-framework/contracts"
+	"github.com/smartcontractkit/chainlink-testing-framework/utils"
+)
+
+var (
+	networkSettings = &blockchain.EVMNetwork{
+		Name:      "geth",
+		Simulated: true,
+		ChainID:   1337,
+		PrivateKeys: []string{
+			"ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
+		},
+		ChainlinkTransactionLimit: 500000,
+		Timeout:                   2 * time.Minute,
+		MinimumConfirmations:      1,
+		GasEstimationBuffer:       10000,
+	}
```

Comment:
is the plan to remove [this](https://github.com/smartcontractkit/chainlink-testing-framework/blob/87a176805b728a0013b381987e67cafe68eb5048/blockchain/config.go#L14) as well  ?